libtransmission/file-posix.c

```diff
@@ -27,6 +28,26 @@
 #include <xfs/xfs.h>
 #endif
 
+/* OS-specific file copy (copy_file_range, sendfile64, or copyfile). */
+#if defined(__linux__)
+#   if defined(HAVE_COPY_FILE_RANGE)
+/* Linux's copy_file_range(2) is buggy prior to 5.3. */
+#       include <linux/version.h>
+#       if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 3, 0)
+#           define USE_COPY_FILE_RANGE
+#       endif
+#   endif /* HAVE_COPY_FILE_RANGE */
+#   if !defined(USE_COPY_FILE_RANGE) && defined(HAVE_SENDFILE64)
+#       define USE_SENDFILE64
+#       include <sys/sendfile.h>
+#   endif
+#elif defined(__APPLE__) && defined(HAVE_COPYFILE)
+#   include <copyfile.h>
```

Comment:
Should we define `USE_COPYFILE` here to be in line with other options and avoid repeating the above condition later on?